### PR TITLE
kubemark: fix start-kubemark.sh

### PR DESCRIPTION
**Pull request by hongchaodeng**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37334)

### CODE REVIEW

**Comment by hongchaodeng** [comment]
cc @gmarek @wojtek-t

File: test/kubemark/start-kubemark.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ ${APISERVER_TEST_ARGS}
 --storage-backend=${STORAGE_BACKEND}
 --service-cluster-ip-range="${SERVICE_CLUSTER_IP_RANGE}"
 EOF
-if [ -z "${CUSTOM_ADMISSION_PLUGINS:-}"]; then
+if [ -z "${CUSTOM_ADMISSION_PLUGINS:-}" ]; then
```
Comment (ixdy) [inline test/kubemark/start-kubemark.sh]: might also want to change this to `[[ ]]` form, as we normally use.

**Review by ixdy — APPROVED** [review]
LGTM with one suggestion

**Comment by ixdy** [comment]
should this be tagged for 1.5?

File: test/kubemark/start-kubemark.sh (hunk 1 above)
Comment (hongchaodeng) [inline test/kubemark/start-kubemark.sh]: Up to @gmarek

**Comment by gmarek** [comment]
Yup - it's a test config fix.

**Comment by hongchaodeng** [comment]
@k8s-bot bazel test this

**Comment by hongchaodeng** [comment]
@k8s-bot bazel test this

**Comment by hongchaodeng** [comment]
@k8s-bot bazel test this

**Pull request merged**